dp/jobqueueing/gcalter-params.py

```diff
@@ -0,0 +1,70 @@
+"""
+Script to alter generate_climos params and PBS params of queue entries.
+Altered entry must be in status NEW.
+"""
+
+from argparse import ArgumentParser
+import logging
+import sys
+
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+from dp.script_helpers import default_logger
+from dp.jobqueueing.argparse_helpers import add_global_arguments, add_generate_climos_arguments, add_pbs_arguments
+from dp.jobqueueing.jobqueueing_db import GenerateClimosQueueEntry
+
+
+logger = default_logger()
+
+
+def update_generate_climos_queue_entries_with_params(session, args):
+    entries = session.query(GenerateClimosQueueEntry)\
+        .filter(GenerateClimosQueueEntry.status == 'NEW')\
+        .all()
+
+    def args_to_entry(attr):
+        """Oh for *#@%* sake. Oughta migrate the database but SQLite doesn't rename columns easily.
```

Comment:
Yeah, given that I wanted to get this working fairly quickly, I decided not to risk getting bogged down in some SQLite rabbit hole (relatively low risk, I now think, but ...).